boards/sim/sim/sim/configs/nsh/defconfig

```diff
@@ -16,6 +16,8 @@ CONFIG_BOARDCTL_POWEROFF=y
 CONFIG_BOARD_LOOPSPERMSEC=0
 CONFIG_BOOT_RUNFROMEXTSRAM=y
 CONFIG_BUILTIN=y
+CONFIG_COVERAGE_ALL=y
+CONFIG_COVERAGE_TOOLCHAIN=y
```

Comment:
Why not add a new config?

Reply:
to save the ci resource. @W-M-R let's move to configs/asan.

Reply:
sim/kasan already has this config

Reply:
okey，My original intention was that sim kasan was unable to directly use this gcov configuration due to the setting of initargs. So I want to make it easier to open the image of NSH